python/extensions/response_stream/_20_live_response.py

```
@@ -30,7 +30,7 @@ async def execute(
                 loop_data.params_temporary["log_item_response"] = (
                     self.agent.context.log.log(
                         type="response",
-                        heading=f"icon://chat {self.agent.agent_name}: Responding",
+                        heading=f"icon://chat Responding",
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)